ballerina/modules/types.wrappers/tests/protobuf_test.bal

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+@test:Config {}
+isolated function testWrappersInt() {
+    int[] intArray = [1, 2, 3, 4];
+    stream<int> intStream = intArray.toStream();
+    stream<int, error?> outputStream = new (new IntStream(intStream));
```

Comment:
Shall we assign the `new IntStream(intStream)` to `IntStream` and use it directly?

Reply:
Updated in 0a4eaf7